subprojects/architecture-test/src/test/java/org/gradle/architecture/test/ArchUnitFixture.java

```diff
@@ -188,6 +189,16 @@ static DescribedPredicate<JavaMember> annotatedMaybeInSupertypeWith(final Descri
         return new AnnotatedMaybeInSupertypePredicate(predicate);
     }
 
+    static ArchCondition<JavaClass> beAnnotatedMaybeInPackageWith(final Class<? extends Annotation> annotationType) {
```

Comment:
```suggestion
    static ArchCondition<JavaClass> beAnnotatedOrInPackageAnnotatedWith(final Class<? extends Annotation> annotationType) {
```

How about this name instead?

Reply:
That sounds good 👍